### ISSUE TITLE
Presentaion

### ISSUE THREAD

**Issue by anander** [jhester/DistributedGA#17]
Just a heads up, I was talking to someone who took this class last semester, and they told me we should have at least 10 slides in our powerpoint.

**Comment by ghost** [comment]
Lets put it together and figure out what we are going to say Tuesday, I'm thinking the presentation is at least 50% of the grade

**Comment by jhester** [comment]
Yeah the presentation is huge.  Do we have any example presentations that we know got an A?
